README.md

```diff
@@ -5,38 +5,67 @@
 
 <p align="center"><img src="assets/sc-timetable.png" style="border-radius: 1rem" width="500"/></p>
 
-## About the project
+# Compa
 
-Compa aims to be the resource-central for higher education. As a fresher, easing into the school system has lots of friction. Finding interest groups/clubs, timetable, starting conversations, learning from seniors and past students is simply difficult.
+A companion application for students to manage and access resources at their higher education institution.
+
+## About
+
+As a fresher, settling into school could be easier. You usually need to find:
+
+- Groups/Clubs you are interested in
```

Comment:
```suggestion
- Communities/Clubs you are interested in
```

Because we'll name that feature "Communities"

> Clubs will be privileged communities